webpages/styles/colors.css

```diff
@@ -1,6 +1,7 @@
 :root {
   --brand-orange: #ff7b26;
   --brand-orange-opacity35: rgba(255, 123, 38, 0.35);
+  --brand-orange-option: rgba(255, 123, 38, 0.7);
```

Comment:
`brand-orange-option` isn't very clear, it should probably be named something like `brand-orange-opacity70`